### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by GORobert**

### CODE REVIEW

**Comment by alserom** [comment]
Hi @GORobert!
Thanks for your PR. Can you add a new patch version to [updates/version.yaml](https://github.com/alserom/octobercms_clearcachewidget/blob/master/updates/version.yaml) file?
`1.3.1: Add Polish translation`
Also, you can add your name to the `Available locales` section in [README.md](https://github.com/alserom/octobercms_clearcachewidget/blob/master/README.md) file.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ You can display a widget with a chart or like a single button. Display options s
 - tr (by [Sevdin Filiz](https://github.com/angelside))
 - nl (by [Alwin Drenth](https://github.com/adrenth))
 - sk (by [Marek Závacký](https://github.com/emzet))
+- sk (by [Robert Pakszys](https://github.com/gorobert))
```
Comment (alserom) [inline README.md]: Should be `- pl (by [Robert Pakszys](https://github.com/gorobert))`?

**Review by alserom — CHANGES_REQUESTED** [review]

**Comment by GORobert** [comment]
Yes sorry fot that

File: README.md (hunk 1 above)
Comment (GORobert) [reply README.md]: Yes

**Review by alserom — APPROVED** [review]

**Pull request merged**